egs/csj/asr1/RESULTS.md

```diff
@@ -1,3 +1,10 @@
+# Lightweight/Dynamic convolution results
```

Comment:
Can you upload the pretrained model? It is important because we do not have such a lightweight model now and we can know how lightweight it is.